rotkehlchen/db/schema.py

```diff
@@ -444,6 +446,7 @@
 CREATE TABLE IF NOT EXISTS optimism_transactions (
     tx_id INTEGER NOT NULL PRIMARY KEY,
     l1_fee TEXT,
+    chain_id INTEGER NOT NULL,
```

Comment:
As I said in the other part ...

Ah damn sorry I am an idiot. I just realized after looking at the code again this upgrade step is not needed. `tx_id` references the `identifier` of `evm_transactions`. Which combines tx_hash + chain_id. So chain id is not needed here.